Created a FizzBuzz program

drewesk opened the pull request:
- All numbers divisible by 3 will print 'Fizz'
- All divisible by 5 prints 'Buzz'
- All divisible by both prints 'FizzBuzz'

[#131780283]

jasonnoble commented on arrays/fizz_buzz.rb:
```diff
@@ -0,0 +1,16 @@
+numbers = (1..100).to_a
+numbers.each do |number|
+  if (number % 3).zero? && (number % 5).zero?
+    puts' FizzBuzz '
```
`puts 'FizzBuzz'`

jasonnoble commented on arrays/fizz_buzz.rb:
```diff
@@ -0,0 +1,16 @@
+  if (number % 3).zero? && (number % 5).zero?
+    puts' FizzBuzz '
+  elsif (number % 5).zero?
+    puts ' Buzz '
```
Don't need the spaces around the word...  `puts 'Buzz'`

jasonnoble commented on arrays/fizz_buzz.rb:
```diff
@@ -0,0 +1,16 @@
+  elsif (number % 5).zero?
+    puts ' Buzz '
+  elsif (number % 3).zero?
+    puts ' Fizz '
```
Don't need the spaces around the word...  `puts 'Fizz'`

jasonnoble commented on arrays/fizz_buzz.rb:
```diff
@@ -0,0 +1,16 @@
+  elsif (number % 3).zero?
+    puts ' Fizz '
+  else
+    puts " #{number} "
```
Don't need the spaces around the word...  `puts number`

drewesk committed:
Moved fizz_buzz.rb to the main dir

* Deleted arrays directory

* Modified file based on instructor feedback

[#131780283]

Created a FizzBuzz program

* All numbers divisible by 3 will print 'Fizz'

* All divisible by 5 prints 'Buzz'

* All divisible by both prints 'FizzBuzz'

[#131780283]

Updated to print out number

* Added a line to print all numbers not divisible by 3 or 5.

[#131780283]

WIP

Rearranged order of conditionals to get the FizzBuzz output to work

* Modified until rubo-cop had no offenses

* '(1..100).to_a to get the range of test numbers

* Verified FizzBuzz output

[#131780283]

removed arrays dir

* rm arrays dir

[#131780283]

Removed guard files

* rm Guardfile and .rubocop.yml to avoid updating the files on master

[#131780283]

Fixed bug and replaced interpolation with peram var

* Fixed a typo on line 6

* replaced interpolation with perameter variable on line 12

[#131780283]

Moved fizz_buzz.rb to methods_and_flowcontrol dir

* Deleted old file on PR and replaced with an
 exact file in methods_and_flowcontrol directory

[#131780283]

Deleted the fizz_buzz.rb in main dir

[#131780283]

Reformatted the code for proper indentation

* Used the reformat tool in rubymine.

[#131780283]

jasonnoble commented on fizz_buzz.rb:
```diff
@@ -0,0 +1,16 @@
+numbers = (1..100).to_a
+numbers.each do |number|
+  if (number % 3).zero? && (number % 5).zero?
+    puts'FizzBuzz'
```
`puts 'FizzBuzz'`

jasonnoble commented on fizz_buzz.rb:
```diff
@@ -0,0 +1,16 @@
+  elsif (number % 3).zero?
+    puts 'Fizz'
+  else
+    puts "#{number}"
```
`puts number`

jasonnoble commented:
Tests are pending...  File should be in methods_and_flow_control/fizz_buzz.rb.
